### DISCUSSION TITLE
GP11 Motion detection stops after a while

### DISCUSSION THREAD

**Discussion by Bocio** [gopro/labs#809] [Q&A] [answered]
I'm using a GP 11 with  motion detection.

I disabled power off and left everything as default (repeat motion capture). Everything seems to work but after a while without seeing any motion, the motion detection stops working and I find the camera idle.
I'm in 4K@25p with horizon lock.
I thought it could works until the battery dies.

Any suggestion?

**Comment by dnewman-gpsw** [comment, marked as answer]
After how long?

What does the log file report?  In MISC/qrlog.txt

**Reply by Bocio** [reply]
It was my fault or something like that.
I set "disable auto power off" via the page https://gopro.github.io/labs/control/motion/ but somehow it was not working so I disable power off via the camera menu.
unfortunately I had formatted the sd card already....

I'm trying to use this function to film closely some huge herons on their trees but no luck so far. Time for another question...

Thanks

**Reply by dnewman-gpsw** [reply]
I fixed the "disable auto power off" within the motion trigger pages.  That was the older style command that wouldn't work on newer cameras.

**Reply by Bocio** [reply]
Thanks
